### ISSUE TITLE
Comments sur les fiches machines

### ISSUE THREAD

**Issue by maayaneaha** [Atelier-Epita/fiches-machines#6]
1. Rajouter une photo avec des légendes des boutons/touches énoncé(e)s dans la fiche machine ?
2. Sur chaque fiche machine, il y a un level indiqué -> peut-être expliquer à quoi il correspond (ou changer "level" par autre chose pour que ce soit + clair ?)

**Comment by maayaneaha** [comment]
> Plutot afficher dans le maker a plusieurs endroit ce a quoi ca correspond donc faire une fiche explicative pour ca 🚀

On peut en mettre une par kiosque ?
Comme ça elles sont bien visibles

> Je penses que les photos c'est une bonne idée lorsque les boutons ne sont pas obvious ou alors faire une fiche descriptives des boutons si il y en a trop.

A mon avis, il faudrait en faire une lorsque le level est orange ou plus, pour être surs que les utilisateurs savent quoi faire

**Comment by jubbakka** [comment]
> > Plutot afficher dans le maker a plusieurs endroit ce a quoi ca correspond donc faire une fiche explicative pour ca 🚀
> On peut en mettre une par kiosque ? Comme ça elles sont bien visibles
yes une par kiosque + une sur le tableau d'affichage de l'entré.
> > Je penses que les photos c'est une bonne idée lorsque les boutons ne sont pas obvious ou alors faire une fiche descriptives des boutons si il y en a trop.
> A mon avis, il faudrait en faire une lorsque le level est orange ou plus, pour être surs que les utilisateurs savent quoi faire.
Ca concernera seulement quelque fiches machines.
pour utiliser des machines orange ou plus il faut d'abord une formation avec un staff.

**Comment by maayaneaha** [comment]
> yes une par kiosque + une sur le tableau d'affichage de l'entrée.

Ce qu'on peut faire c'est faire des "personnalisées" en fonction du matériel proposé sur le kiosque. Qu'en penses-tu ?
> Ca concernera seulement quelque fiches machines. Pour utiliser des machines orange ou plus il faut d'abord une formation avec un staff.

Yes, on peut aussi en faire une et la laisser en "rappel" on ne sait jamais, il vaut mieux avoir trop de rappels plutôt que pas assez.
